po/de.po

```diff
@@ -377,7 +377,7 @@ msgstr "SCAVENGER-BASIS ENTDECKT"
 
 #: data/base/messages/strings/cam1strings.txt:146
 msgid "Defeat the scavengers and retrieve any artifacts."
-msgstr "Vernichten Sie die Scavengers und bergen Sie alle Artefakte."
+msgstr "Vernichten Sie die Scavenger und bergen Sie alle Artefakte."
```

Comment:
:x:

Copying my reply to this from Trac ticket 4845:
> I am seeing this similar to the (now also) German word "Hobby": We capitalize the noun, but we use the original plural. The translation stays as it is. This is also true for similar cases.
> 
> Side note: Actually, in German, we do not use the original plural for this word, although the original plural is usually used in foreign words such as "Tempus", "Genus", "Epitheton" or "Nomen". Instead, officially correct German uses (only) "Hobbys".

Reply:
Since Trac did not allow my reply to posted in full, I uploaded it to [Pastebin.com](https://pastebin.com/raw/HBPi5Rnx).

Reply:
> Since Trac did not allow my reply to posted in full, I uploaded it to [Pastebin.com](https://pastebin.com/raw/HBPi5Rnx).

Thanks.
The file is archived at https://archive.fo/Da02A to prevent link rot.

Reply:
> x
> 
> Copying my reply to this from Trac ticket 4845:
> 
> > I am seeing this similar to the (now also) German word "Hobby": We capitalize the noun, but we use the original plural. The translation stays as it is. This is also true for similar cases.
> > Side note: Actually, in German, we do not use the original plural for this word, although the original plural is usually used in foreign words such as "Tempus", "Genus", "Epitheton" or "Nomen". Instead, officially correct German uses (only) "Hobbys".

My point is that we translate "scavengers" inconsistently.
In one instance its plural form ends in an s, unlike in all others:

```
msgid "Defeat the scavengers and retrieve any artifacts."
msgstr "Vernichten Sie die Scavengers und bergen Sie alle Artefakte."

msgid "Scavengers"
msgstr "Scavenger"

msgid "No Scavengers"
msgstr "Keine Scavenger"
```

Reply:
> My point is that we translate "scavengers" inconsistently.
> In one instance its plural form ends in an s, unlike in all others:
> 
> ```
> msgid "Defeat the scavengers and retrieve any artifacts."
> msgstr "Vernichten Sie die Scavengers und bergen Sie alle Artefakte."
> 
> msgid "Scavengers"
> msgstr "Scavenger"
> 
> msgid "No Scavengers"
> msgstr "Keine Scavenger"
> ```

I see. Do you have statistics on which variant prevails? There should definitely be consistency.

> That's why I limited my suggestions to the few translations that could
> be easily misunderstood and accepted all your other disagreements.

Okay, that's reasonable. Next time, please mention this somewhere in advance :heart:

Reply:
> > My point is that we translate "scavengers" inconsistently.
> > In one instance its plural form ends in an s, unlike in all others:
> > ```
> > msgid "Defeat the scavengers and retrieve any artifacts."
> > msgstr "Vernichten Sie die Scavengers und bergen Sie alle Artefakte."
> > 
> > msgid "Scavengers"
> > msgstr "Scavenger"
> > 
> > msgid "No Scavengers"
> > msgstr "Keine Scavenger"
> > ```
> 
> I see. Do you have statistics on which variant prevails? There should definitely be consistency.
> [...]

According to Linguee, the variant without s prevails:
https://www.linguee.de/deutsch-englisch/search?source=englisch&query=scavengers

Reply:
I meant our translation, not in general. I just ran `grep --after=1 -E -i '^msgid.*scavenger' po/de.po` and saw that you already dug up all occurrences of the plural.

I'm convinced! Your translation is better. :)

Reply:
> I don't think that there are "statistics" which variant prevails. As wrote in my comment to [#326](https://github.com/Warzone2100/warzone2100/pull/326) I see this similar to the Italian word "Espresso". This is now also used in German with the original plural "Espressi". So I'm for the translation "Scavengers" in all places where necessary.

Linguee did not find a single instance where "scavengers" was translated as "Scavengers".

Reply:
I accept your decision

Reply:
> Linguee did not find a single translation of "scavengers" as "Scavengers".

Note: This is not relevant here. "New Paradigm" and "Scavenger(s)" were used as faction _names_ in the original German translation. That is also the reason why the names were not translated.

Reply:
In my opinion the developer would name them "Neues Leitbild" and "Plünderer" if it was a german developer.
(I don't like anglicisms.)

Reply:
The German videos were made by Pumpkin Studios/Eidos Interactive and these factions are called "Scavenger(s)" and "New Paradigm" there.

Just a few examples:
* `cam1ascv.ogg`: "Überfallkommandos der Scavengers nähern sich unserer Basis. Verteidigen Sie die Basis und zerstören Sie anschließend die Lager der Scavengers."
* `cam1ca.ogg`: "Commander, Sie haben das New Paradigm aus dieser Zone vertrieben. [...]"
* `cam1cnp.ogg`: "Eindringling, der unrechtmäßige Angriff auf unsere friedlichen Einrichtungen wird nicht ungestraft bleiben. Sie verstoßen gegen die Rechte des New Paradigm und werden zerstört werden."

This leaves no room to change the faction names.